src/components/v5/shared/Navigation/ColonySwitcher/ColonySwitcher.tsx

```diff
@@ -47,9 +47,12 @@ const ColonySwitcher: React.FC<ColonySwitcherProps> = ({
               enableMobileAndDesktopLayoutBreakpoints
             }
           />
-          {colonyContext?.colony.name && (
+          {colonyContext?.colony && (
             <p className="truncate text-md font-semibold text-base-white">
-              {capitalizeFirstLetter(colonyContext.colony.name)}
+              {capitalizeFirstLetter(
+                colonyContext.colony.metadata?.displayName ??
+                  colonyContext.colony.name,
+              )}
             </p>
```

Comment:
I would propose to create a constant 
`const colonyName = colonyContext?.colony.metadata?.displayName ?? colonyContext?.colony.name`
and then refactor these lines as
```
{colonyName && (
   <p className="truncate text-md font-semibold text-base-white">
       {capitalizeFirstLetter(colonyName)}
   </p>
 )}
```

Reply:
All right lemme do that 👌